src/app/pages/BlocksPage/index.tsx

```diff
@@ -1,21 +1,34 @@
-import { FC } from 'react'
+import { FC, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { AxiosResponse } from 'axios'
 import Divider from '@mui/material/Divider'
 import useMediaQuery from '@mui/material/useMediaQuery'
-import { useTheme } from '@mui/material/styles'
+import { styled, useTheme } from '@mui/material/styles'
 import { PageLayout } from '../../components/PageLayout'
 import { SubPageCard } from '../../components/SubPageCard'
 import { Layer, useGetRuntimeBlocks } from '../../../oasis-indexer/api'
 import { Blocks, TableRuntimeBlockList } from '../../components/Blocks'
 import { NUMBER_OF_ITEMS_ON_SEPARATE_PAGE, REFETCH_INTERVAL } from '../../config'
 import { useSearchParamsPagination } from '../../components/Table/useSearchParamsPagination'
+import { BlockDetailView } from '../BlockDetailPage'
+import Box from '@mui/material/Box'
+import { COLORS } from '../../../styles/theme/colors'
 import { AppErrors } from '../../../types/errors'
 import { useLayerParam } from '../../hooks/useLayerParam'
+import { TableView, TableViewSpeedDial } from '../../components/TableViewSpeedDial'
+import { LoadMoreButton } from '../../components/LoadMoreButton'
 
 const PAGE_SIZE = NUMBER_OF_ITEMS_ON_SEPARATE_PAGE
 
+const BlockDetails = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: `0 ${theme.spacing(2)}`,
+  backgroundColor: COLORS.persianBlue,
```

Comment:
My head hurts

So we have
```tsx
      <SubPageCard[1]
          <BlockDetails
             {blocks.map
                <SubPageCard[2]
                  <BlockDetailView
                    <StyledDescriptionList
```

But `SubPageCard[1]` doesn't have card padding; and `BlockDetails` erases card's background, so it doesn't have card background either! That way `SubPageCard[2]` can add card styling for each block individually instead. But it doesn't have card padding; instead `StyledDescriptionList` adds card padding.  That way `StyledDescriptionList` can override `SubPageCard[2]`'s background for new blocks.